examples/basic.rs

```diff
@@ -82,6 +82,16 @@ async fn main() -> Result<()> {
         }
     }
 
+    println!("Queries requested: {}", session.queries_num());
+    println!("Iter queries requested: {}", session.queries_iter_num());
+    println!("Errors occured: {}", session.errors_num());
+    println!("Iter errors occured: {}", session.errors_iter_num());
+    println!("Average latency: {}", session.latency_avg());
+    println!(
+        "87,1 latency percentile: {}",
+        session.latency_percentile(87.1)
```

Comment:
Let's show `latency_percentile(99.9)` in the example -- that's a very common metric to check.